CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+
+Therefore we want to create strong community of contributors -- all working together to create the kind of delightful experience that our developers deserve.
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given. ❤️
+
+# Types of Contributions
+
+## Report bugs
+
+Report bugs at https://github.com/spotify/backstage/issues.
+
+## Fix bugs or build new features
+
+Look through the GitHub issues for [bugs](https://github.com/spotify/backstage/labels/bugs), [good first issues](https://github.com/spotify/backstage/labels/good%20first%20issue) or [help wanted](https://github.com/spotify/backstage/labels/help%20wanted).
+
+## Build a Plugin
+
+The value of Backstage grows with every new Plugin that gets added. Wouldn't it be fantastic if there was a Plugin for every infrastructure projects out there? We think so.
+
+What kind of Plugins should/could be created? Some inspiration from the 120+ Plugins that we have developed inside Spotify can be found [here](https://backstage.io/demos), but the we will keep a running list of suggestions labeled with `[plugin]`:
+
+https://github.com/spotify/backstage/labels/plugin
+
+A great reference example of a Plugin can be found here https://backstage.io/blog/2020/04/06/lighthouse-plugin (thanks [@fastfrwrd](https://github.com/fastfrwrd)!)
+
+## Write Documentation
+
+The current documentation is very limited. Help us make the `/docs` folder come alive.
+
+## Contribute to Storybook
+
+We think the best way to ensure different Plugins provide a consistent experience is through a solid set of reusable UI/UX components. Backstage uses [Storybook](http://storybook.backstage.io).
+
+Either help us [create new components](https://github.com/spotify/backstage/labels/help%20wanted) or improve stories for the existing ones (look for files with `*.stories.tsx`).
+
+## Submit Feedback
+
+The best way to send feedback is to file an issue at https://github.com/spotify/backstage/issues.
+
+If you are proposing a feature:
+
+- Explain in detail how it would work.
+- Keep the scope as narrow as possible, to make it easier to implement.
+- Use appropriate labels
+- Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+# Get Started!
+
+So...feel ready to jump in? Let's do this. Head over to the [Getting Started guide](https://github.com/spotify/backstage#getting-started) 👏🏻💯
+
+If you need help, just jump into our [Discord chatroom](https://discord.gg/MUpMjP2).
+
+# Code of Conduct
+
+This project adheres to the [Spotify FOSS Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
+
+[code-of-conduct]: https://github.com/spotify/backstage/blob/master/CODE_OF_CONDUCT.md
```

Comment:
# Security Issues?

Please report sensitive security issues via Spotify's bug-bounty program (https://hackerone.com/spotify) rather than GitHub.

Reply:
Fixed. Thanks.